RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -30,6 +30,7 @@ boolean acquireGumItem(item it);
 boolean acquireHermitItem(item it);
 boolean isHermitAvailable();
 boolean isGeneralStoreAvailable();
+boolean isMusGuildStoreAvailable();
```

Comment:
not necessary.

Reply:
Why is it not necessary? I thought all functions need to have a public prototype, and as far as I can tell all of them do.

Reply:
That's what autoscend_header.ash is for. Whoever started declaring functions at the top of files didn't understand what it's for. The only time you need to do that is to avoid circular dependencies. Otherwise this is the very definition of cargo cult programming.

Reply:
I see, thank you for explaining it.
also, change made.